src/common/low_precision_transformations/src/convolution.cpp

```diff
@@ -170,9 +170,11 @@ bool ConvolutionTransformation::transform(TransformationContext &context, ngraph
         }
 
         const auto copyNode = convolution->clone_with_new_inputs({ dequantization.multiply->input_value(0), convolution->input_value(1) });
-        auto conv = ov::as_type_ptr<opset1::Convolution>(copyNode);
+        copyNode->set_friendly_name(convolution->get_friendly_name());
+        ngraph::copy_runtime_info(convolution, copyNode);
```

Comment:
@GlebKazantaev, does `ov::copy_runtime_info` copy input/output port runtime attributes like as `NetworkHelper::copyInfo`? Should we use more complex `NetworkHelper::copyInfo` here?

Reply:
copy_runtime_info working only with nodes. 
Discussed offline with @v-Golubev that such replacements can be simplified and do not require rt info copy.